src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+    "description": "Thanks for trying out Cookie AutoDelete. If you liked it, then please give a Review."
+  },
+
+  "releaseNotesText": {
+    "message": "Versionshinweise",
+    "description": "Release Notes"
+  },
+
+"activeModeText": {
+    "message": "Aktiven Modus einschalten? Verzögerung bis zum Aufräumen:",
+    "description": "Enable Active Mode? Delay Before Cleaning:"
+  },
+
+"minutesText": {
+    "message": "Minute(n)",
+    "description": "Minute(s)"
+  },
+
+"activeModeTooltipText": {
+    "message": "Wenn der aktive Modus eingeschaltet ist wird das Aufräumen der Cookies beim Schließen eines Tabs ausgeführt",
+    "description": "When active mode is enabled, cookie cleanup is triggered on tab close."
+  },
+
+"statLoggingText": {
+    "message": "Gesamtanzahl gelöschter Cookies protokollieren",
+    "description": "Log Total Number Of Cookies Deleted"
+  },
+
+"statLoggingTooltipText": {
+    "message": "Zählt die Anzahl der gelöschten Cookies, wird im Willkommensbildschirm angezeigt",
+    "description": "Counts the number of cookies deleted during this session and in total. This is shown in the Welcome Screen."
+  },
+
+"resetCounterText": {
+    "message": "Zähler zurücksetzen",
+    "description": "Reset Counter"
+  },
+
+"showNumberOfCookiesInIconText": {
+    "message": "Zeigt die Anzahl der Cookies einer Domain in der Symbolleiste",
+    "description": "Show Number of Cookies for that Domain"
+  },
+
+"showNumberOfCookiesInIconTooltipText": {
+    "message": "Zeigt die Anzahl der Cookies einer Domain im Tooltip des Symbols in der Symbolleiste",
+    "description": "Show Number of Cookies for that Domain"
+  },
+
+"notifyCookieCleanUpText": {
+    "message": "Zeigt eine Benachrichtungen nach dem Aufräumen der Cookies",
+    "description": "Show Notification After Cookie CleanUp"
+  },
+
+"notifyCookieCleanUpTooltipText": {
+    "message": "Zeigt wie viele Cookies für welche Domains gelöscht wurden.",
+    "description": "Shows how many cookies were cleaned and for which domain."
+  },
+
+"contextualIdentitiesEnabledText": {
+    "message": "Aktiviert die Unterstützung für Firefox' \"Contextual Identities\" (EXPERIMENTELL Firefox 53+)",
+    "description": "Enable Support for Firefox's Contextual Identities (EXPERIMENTAL Firefox 53+)"
+  },
+
+"contextualIdentitiesTooltipText": {
+    "message": "NICHT aktivieren solange du nicht Firefox 53+ nutzt. Fügt eine Whitelist für JEDEN Container hinzu, den du nutzt. Die Standard-Whitelist bleibt davon unabhängig.",
+    "description": "Do NOT Enable unless you are on Firefox 53+. This will add a different whitelist for EACH containers you use. Your default whitelist is separate from these."
+  },
+
+"saveText": {
+    "message": "Speichern",
+    "description": "Save"
+  },
+
+"cancelText": {
+    "message": "Abbrechen",
+    "description": "Cancel"
+  },
+
+"defaultSettingsText": {
+    "message": "Standardeinstellungen",
+    "description": "Default Settings"
```

Comment:
Usually I'd suggest "Restore default settings" (and "Standardeinstellungen wiederherstellen") here, but I didn't want to change the meaning too much.